app/classifier.py

```diff
@@ -219,9 +219,10 @@ def _classify_probs(self, seqs: List[str]) -> List[Dict[str, float]]:
             return []
         # Split very large requests into chunks since
         # (intermediate) bert data is huge.
-        if len(seqs) > 1000:
-            return (self._classify_probs(seqs[:1000]) +
-                    self._classify_probs(seqs[1000:]))
+        BATCH_SIZE = 200
```

Comment:
please put constants at the top of the file

Reply:
I changed the batch size from being a constant to an optional argument.
This way it's easier to adapt it from a higher level.